system-variables.md

```diff
@@ -2451,6 +2451,17 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 类型：字符串
 - 这个变量用来设置基于负载的 replica read 的触发阈值。当 leader 节点的预估排队时间超过阈值时，TiDB 会优先从 follower 节点读取数据。格式为时间，例如 `"100ms"` 或 `"1s"`。详情见 [TiDB 热点问题处理](/troubleshoot-hot-spot-issues.md#打散读热点)。
 
+### `tidb_lock_unchanged_keys` <span class="version-mark">从 v7.3.0/v7.1.1 版本开始引入</span>
```

Comment:
```suggestion
### `tidb_lock_unchanged_keys` <span class="version-mark">从 v7.1.1 版本开始引入</span>
```

Reply:
That's inaccurate. It's not in 7.2.0.

Reply:
The label "New in v7.1.1" indicates that this variable was introduced after v7.1.1. Given that v7.2.0 was released before v7.1.1, (see [release timeline](https://docs.pingcap.com/tidb/dev/release-timeline)), this variable doesn't exist in v7.2.0.

To avoid any confusion about whether this variable is available in v7.2.0, we can switch the base of this PR to release-7.1 and only add this variable to the v7.1 documentation.

After the code freeze of v7.3.0, we can cherry-pick this PR to the master branch and update the description to "New in v7.1.1 and v7.3.0".

Reply:
@ekexium PTAL

Reply:
@ekexium I have created the PR https://github.com/pingcap/docs-cn/pull/14508 and https://github.com/pingcap/docs/pull/14222 for v7.1.1, PTAL